app-router/localization.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Localization
+description: "This guide walks you through setting up localization for your Makeswift site so you can visually customize your site for different locales."
+---
+
+import AppCatchAllLocalizedExample from "/snippets/app-router/catch-all-localized.mdx"
+import LocalizationExample from "/snippets/app-router/localization.mdx"
+import NextConfigLocalizedExample from "/snippets/reference/next-config-localized.mdx"
+import NextConfigDomainLocalizedExample from "/snippets/reference/next-config-domain-localized.mdx"
+
+<Info>
+  The feature is deeply integrated with [Next.js' internationalization
+  features](https://nextjs.org/docs/app/building-your-application/routing/internationalization).
+</Info>
+
+<Warning>
+  Localization is currently only available on the Enterprise plan. Reach out to
+  support at [support@makeswift.com](mailto:support@makeswift.com) if you want
+  to enable this feature.
+</Warning>
+
+## Getting started
+
+<Steps>
+
+<Step title="Add locales in the Makeswift builder">
+Open site settings and go to the "Locales" tab:
+
+![Localization first step](../images/localization-first-step.png)
+
+To add a new locale, click the "+ Add locale" button. You can modify or delete existing locales by hovering over the locale:
+
+![Hovering over the locale](../images/localization-hover-locale.png)
+
+You can also modify the default locale by hovering over the default locale and clicking the edit button.
+
+Once you add all the locales you need, it might look like this:
+
+![Manage locales in settings](../images/localization-manage-locales.png)
+
+</Step>
+
+<Step title="Configure locales in your Next.js config">
```

Comment:
(Blocking) You no longer have to configure anything in the next config for localization.

I might instead make a super short section with a title and a link telling them to read the Internationalization docs for app router, to make sure that they have the `[lang]` segment in their catchall route.

Reply:
✅